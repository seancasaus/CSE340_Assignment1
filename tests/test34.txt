MtwE6 (k).(((R)|(c))|((((M).(E)).((s)|(V))).(N))) , Mx d , nb5UG F , kMYJE (9)|((l).((J)|(7))) , wUPFO (N)|((p).((f).((c)|((o)|(V))))) , eZFQc ((((U)|(B)).(x)).((r)|(z))).(((U)|(h))|((((U).(E)).((_).(6)))|(A))) , pIyJt (((I)|(((K).(B))|((Q).(Q)))).(y))* , YHiGS ((((v).(S))|(2)).(T)).(K) , Pe92q Z , bji2c ((a)|(((f).(w)).((r)|((L)|(a))))).((o).(a)) #
" ZKByKByNpfc2TKFIyQQyUxrAfwroa2TKFl7dQQyQQyaoapfVBxrUUxrh2TK2TKUxrUE6NNpfcpfcvSTKl72TKNkckMEsNZUxrAvSTKFBxrhd2TKF2TK2TKdaoaF9vSTKIyQQyUxrhUxrUE6ZvSTKFNpfVNZkRfwroakcaoafwaoapfcaoaBxrUddpfc " 