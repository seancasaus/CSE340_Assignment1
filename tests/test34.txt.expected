Pe92q , "Z"
pIyJt , "KByKBy"
wUPFO , "N"
wUPFO , "pfc"
YHiGS , "2TK"
nb5UG , "F"
pIyJt , "IyQQy"
eZFQc , "UxrA"
bji2c , "fwroa"
YHiGS , "2TK"
nb5UG , "F"
kMYJE , "l7"
Mx , "d"
pIyJt , "QQyQQy"
bji2c , "aoa"
wUPFO , "pfV"
eZFQc , "BxrU"
eZFQc , "Uxrh"
YHiGS , "2TK"
YHiGS , "2TK"
eZFQc , "UxrUE6"
wUPFO , "N"
wUPFO , "N"
wUPFO , "pfc"
wUPFO , "pfc"
YHiGS , "vSTK"
kMYJE , "l7"
YHiGS , "2TK"
wUPFO , "N"
MtwE6 , "kc"
MtwE6 , "kMEsN"
Pe92q , "Z"
eZFQc , "UxrA"
YHiGS , "vSTK"
nb5UG , "F"
eZFQc , "Bxrh"
Mx , "d"
YHiGS , "2TK"
nb5UG , "F"
YHiGS , "2TK"
YHiGS , "2TK"
Mx , "d"
bji2c , "aoa"
nb5UG , "F"
kMYJE , "9"
YHiGS , "vSTK"
pIyJt , "IyQQy"
eZFQc , "Uxrh"
eZFQc , "UxrUE6"
Pe92q , "Z"
YHiGS , "vSTK"
nb5UG , "F"
wUPFO , "N"
wUPFO , "pfV"
wUPFO , "N"
Pe92q , "Z"
MtwE6 , "kR"
bji2c , "fwroa"
MtwE6 , "kc"
bji2c , "aoa"
bji2c , "fwaoa"
wUPFO , "pfc"
bji2c , "aoa"
eZFQc , "BxrU"
Mx , "d"
Mx , "d"
wUPFO , "pfc"
